app/models/piece.rb

```diff
@@ -8,6 +8,8 @@ class Piece < ActiveRecord::Base
   def move_to!(x, y)
     destination_piece = game.pieces.find_by(x_coordinate: x, y_coordinate: y)
     destination_piece.delete if destination_piece.present?
+    next_turn = Game.find(game_id).turn == 'white' ? 'black' : 'white'
```

Comment:
since the `piece` belongs_to a game. You can use `game.turn` here. Rails takes care of the find for you.